src/backgrounds/_base.scss

```diff
@@ -0,0 +1,113 @@
+/**
+ * @file Backgrounds
+ * @module Backgrounds
+ * @overview Background and matching text styles for page modules
+ * @todo: remove _backdrops.scss since this replaces it?
+ * @todo: rename TrialWarranty to be generic, update to use Starry Background
```

Comment:
:thought_balloon: I was thinking about Trial Warranty on Monday after reviewing the callout pattern. It's basically a callout. Only difference is that it is defined. In fact we have other defined callouts, another example that comes to mind is the newsletter signup half callout/block. 

I'm starting to think that it may behoove us to avoid having too many defined modules in core. I would prefer to focus on providing generic layouts, modules, and element combinations that we can use to compose larger patterns. 

I raise this thought because I wonder if the concept of the Trial Warranty as its own pattern is one we want to keep. I can see it being its own pattern if it always has certain unique styling, say for the icon at different breakpoints, but if it just composes itself off existing patterns with little unique CSS, maybe we don't need to call it out.

</thoughts>